spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/stores/FormStore.java

```diff
@@ -0,0 +1,131 @@
+package com.boundlessgeo.spatialconnect.stores;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.boundlessgeo.spatialconnect.SpatialConnect;
+import com.boundlessgeo.spatialconnect.config.SCFormConfig;
+import com.boundlessgeo.spatialconnect.config.SCStoreConfig;
+import com.boundlessgeo.spatialconnect.dataAdapter.GeoPackageAdapter;
+import com.boundlessgeo.spatialconnect.geometries.SCSpatialFeature;
+import com.boundlessgeo.spatialconnect.query.SCQueryFilter;
+import com.boundlessgeo.spatialconnect.services.SCConfigService;
+import com.boundlessgeo.spatialconnect.services.SCNetworkService;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import rx.Observable;
+import rx.functions.Action0;
+
+/**
```

Comment:
@georific can you add the license to the top of the file before the package statement?